packages/oranger/oranger.2.0.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+authors: ["Francois Berenger"]
+homepage: "https://github.com/UnixJunkie/oranger"
+bug-reports: "https://github.com/UnixJunkie/oranger/issues"
+dev-repo: "git+https://github.com/UnixJunkie/oranger.git"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+install: [
+  ["sh" "install_ranger.sh"]
+]
+depends: [
+  "dune"
+  "conf-cmake" {build}
+  "conf-wget" {build}
+  "re"
+  "dolog"
```

Comment:
```suggestion
  "dolog" {>= "4.0.0"}
```
I'm guessing this requires at least dolog 4.0.0 since anything would be incompatible otherwise?